### PR TITLE
Use read_utf8() and write_utf8()

### LINKED ISSUE #234: Encoding issues in Hebrew. Lost in translation
```r
reprex::reprex("א")
>Rendering reprex...
>Error in gsub("(?<=\n)(?=.|\n)", continue, x, perl = TRUE) : 
 > input string 1 is invalid UTF-8
```

@krlmlr 
This is on a Windows 10 machine with Hebrew language

```r
Sys.getlocale()
> [1] "LC_COLLATE=Hebrew_Israel.1255;LC_CTYPE=Hebrew_Israel.1255;LC_MONETARY=Hebrew_Israel.1255;LC_NUMERIC=C;LC_TIME=Hebrew_Israel.1255"
```

### PR DESCRIPTION
**Pull request by krlmlr**
from _xfun_, which is a dependency anyway via _rmarkdown_ -> _knitr_. This fixes encoding problems on non-Latin locales.

Was meant to address #234, but can't fix it unfortunately.

### CODE REVIEW

**Comment by jennybc** [comment]
What's the status of this PR @krlmlr? Is "WIP" still accurate or should it be considered "ready to go"?

**Comment by krlmlr** [comment]
I think using `read_utf8()` etc. is better than the current implementation, but it won't fix the bug that triggered this PR.

Removed the AppVeyor cache to fix the broken build, also turned on binary installation. Let's see what the Windows test shows.

**Comment by krlmlr** [comment]
I guess we're good to go here.

**Comment by jennybc** [comment]
OK I will experiment with this personally over on Windows.

File: appveyor.yml [hunk 1]
```diff
@@ -13,9 +13,7 @@ install:
 environment:
   global:
     USE_RTOOLS: true
-
-cache:
-  - C:\RLibrary
+    PKGTYPE: binary
```
Comment (jennybc) [inline appveyor.yml]: Shall I roll this back now that we've rebuilt the cache?

**Comment by jennybc** [comment]
I can imagine we'll need refinements. And I'd like to add tests that means something. But this seems like (the first) step in the right direction. Merging.

**Comment by krlmlr** [comment]
Thanks. I don't think the caching on AppVeyor is currently worth its price, especially if installation from binary is enabled. If problems occur again, we can reapply 03c0d5aa6f4b6c7825af7238f588b5d5b6374b56.

**Pull request merged**